sources/documents/3_PB/Verbali/Interni/VerbaleInterno_2024-08-19/todo.tex

```diff
@@ -0,0 +1,40 @@
+\section{Todo / In Progress}\label{sec:todo}
+
+\bgroup
+\begin{center}
+  \begin{longtable}{|P{1.9cm}|P{5.2cm}|P{3cm}|>{\arraybackslash}P{2.4cm}|}
+    \hline
+    \textbf{Ticket} & \textbf{Incarico} & \textbf{Incaricato/a} & \textbf{Scadenza}\\
+    \hline
+    \endfirsthead
+
+    \hline
+		\textbf{Ticket} & \textbf{Incarico} & \textbf{Incaricato/a} & \textbf{Scadenza} \\
+		\hline
+		\endhead
+
+    \multicolumn{4}{|r|}{{Continua nella prossima pagina}} \\ 
+		\hline
+		\endfoot
+
+    \hline
+		\endlastfoot
+    
+    ARGO-270 & Implementazione test \glossario{back-end} generazione \glossario{prompt} & \marco & 2024-08-21 \\
```

Comment:
In questa riga potremmo anche mettere direttamente "Implementazione test back-end relativi a txtai"